snappy_wrappers/wrappers/pvactools/combining/wrapper.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+"""
+Wrapper for preparation the vcf file for somatic neoepitope prediction
+"""
+
+import os
+
+from snakemake.shell import shell
+
+__author__ = "Pham Gia Cuong"
+__email__ = "pham.gia-cuong@bih-charite.de"
+
+step = snakemake.config["pipeline_step"]["name"]
+config = snakemake.config["step_config"][step]
```

Comment:
If you have the time, I'd be happy if all the config accesses in the wrapper here could be expressed as snakemake.params instead, to make the wrapper independent of the configuration file (and any changes to the configuration structure would then only need changes in the params section)

Reply:
E.g.:
`exp_format = snakemake.params["exp_format"]` (basically with `params: exp_format=wf.config["preparation"]["format"]` in the Snakefile [or delegate to the get_params/get_args snappy functions])